client/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "esbuild ./target/scala-2.13/client-opt/main.js --bundle --outdir=../server/src/main/assets/ --minify"
   },
   "dependencies": {
-    "@drt/drt-react": "UKHomeOffice/drt-react#1.7.4",
+    "@drt/drt-react": "UKHomeOffice/drt-react#1.7.5",
```

Comment:
I just noticed this release doesn't exist. The latest release is 1.7.1: https://github.com/UKHomeOffice/drt-react/releases